File: asgn2/python_code/task3_python/Chair.py
# this is the python chair code
import Attendant
import DumbFixture
import Room
import SmartFixture

class Chair(DumbFixture):
    def __init__(self,Name,Weight,Position,Person_in_charge,Seat_back):
        super(Chair,self).__init__(Name,Weight,Position,Person_in_charge)
        self._seat_back=Seat_back

File: Programming_Languages/asgn2/python_code/task3_java/Chair.java
package assg2task3;

/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

/**
 *
 * @author kenchan
 */
public class Chair extends DumbFixture {
    protected Boolean seat_back;
    public Chair(String Name, double Weight, int Position, Attendant Person_in_charge, Boolean Seat_back){
        super(Name, Weight, Position, Person_in_charge);
        this.seat_back = Seat_back;
    }
}
